sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Customized/Models/DataProtectionOperationJobExtendedInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    public partial class DataProtectionOperationJobExtendedInfo
+    {
+        /// <summary> Arm Id of the job created for this operation. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Please add
[Obsolete("This property is obsolete and will be removed in a future release", false)]

Reply:
Fixed